Tests/FileService_test.py

```diff
@@ -3,7 +3,9 @@
 from pathlib import Path
 
 from TM1py import TM1Service
-from .Utils import skip_if_insufficient_version
+from TM1py.Exceptions import TM1pyVersionException
+from .Utils import skip_if_insufficient_version, verify_version
+from .Utils import skip_if_deprecated_in_version as skip_if_version_is_at_least
```

Comment:
We can rename these functions to `skip_if_version_lower_than` and `skip_if_version_higher_than`

This would be clearer than `skip_if_insufficient_version`

Reply:
Thanks @MariusWirtz for the feedback. Yes I agree. These names are better :-) To be precise, I would name the second function `skip_if_version_higher_or_equal_than`. The update is on the way...

Reply:
Done!